resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -5,10 +5,12 @@ restartfile=<%= EZBake::Config[:restart_file] %>
 timeout=<%= EZBake::Config[:start_timeout] %>
 
 if [ -r "$restartfile" ];  then
-    kill -HUP $(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
+    pid=$(pgrep -f <%= EZBake::Config[:uberjar_name] %>)
+    kill -HUP $pid
     cur=$(head -n 1 "$restartfile")
     initial=$cur
     while [ "$cur" == "$initial" ] ;do
+        kill -0 $pid >/dev/null 2>&1
```

Comment:
so `kill -0 $pid` will return non-zero if the process died for some reason, which will then halt the reload script (because of `set -e`. Should we print some error out in that case, or do we assume the application log will have the relevant details?